src/sort.ts

```diff
@@ -1,16 +1,20 @@
-import importSort from 'import-sort';
-import { getConfig } from 'import-sort-config';
-import { dirname, extname } from 'path';
+import { DEFAULT_CONFIGS, getConfig } from 'import-sort-config';
 import { TextDocument, window } from 'vscode';
-import onSave from './on-save';
+import { dirname, extname } from 'path';
 import { getConfiguration, getMaxRange } from './utils';
 
+import importSort from 'import-sort';
+import onSave from './on-save';
```

Comment:
Please don't change sorting style of a single file can you restore the sorting for this file for this PR? If you want to change import sorting let's do it on a different PR to keep changes clear.

Reply:
Totally agree - the extension did it but I will fix it!